tests/integration/test_wild.py

```diff
@@ -80,13 +81,25 @@ def test_amazon_doctype(tmpdir):
     assert 'html' in r.text
 
 
-def test_xmlrpclib(tmpdir):
+@pytest.yield_fixture(scope='session')
+def rpc_server():
```

Comment:
what's this for?

Reply:
It's to remove dependency on bugs.python.org/xmlrpc which has a 301 Moved Permanently.

> Also includes @samuelfekete's CI fixes from 36b5162 to (mostly) get the CI green!